Adding platforms and channel doc section

erickzanardo opened the pull request:
# Description

Adds a simple section on docs to talk about supported platforms and channel

This PR is targeting master as this is just documentation and is already valid for the current version, I will update develop and v1 once this is merged.

spydon commented on doc/platforms.md:
```diff
@@ -0,0 +1,28 @@
+# Supported platforms
+
+Flame runs on top of Flutter, so its supported platforms depended on how mature/stable that platform is supported by Flutter.
```
Depended -> depend
That platform is supported by -> support for each platform is by

spydon commented on doc/platforms.md:
```diff
@@ -0,0 +1,28 @@
+
+## Flutter channels
+
+Flame keeps it support on the stable and beta channel, dev and master channel should work, but we don't support them, this mean that issues happening outside stable or beta channel, are not a priority.
```
Split this up into more sentences (too many commas).

spydon commented on doc/platforms.md:
```diff
@@ -0,0 +1,28 @@
+
+## Flame web
+
+To use Flame on web you need to be on the beta channel of Flutter and the game should be ran using skia rendering, otherwise weird side effects may happen.
```
Ran -> running

spydon commented on doc/platforms.md:
```diff
@@ -0,0 +1,28 @@
+
+`$ flutter run -d Chrome`
+
+And well the app starts and the debug is setup, press the `k` key. that will enable the `CanvasKit` mode.
```
Well -> when

spydon replied:
. that -> , that
Or capital T

luanpotter commented on doc/platforms.md:
```diff
@@ -0,0 +1,28 @@
+# Supported platforms
```
nit: titles in Title Case

erickzanardo commented:
@spydon @luanpotter @renancaraujo I have fixed our lint, apparently the flutter format command now printed some info about collecting usage data, and that was breaking our script, I changed so we check on the exit code instead of checking if the command had printed anything.
